tests/tasks/test_juniperalarmtask.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from zino.config.models import PollDevice
+from zino.events import AlarmEvent
+from zino.state import ZinoState
+from zino.tasks.juniperalarmtask import JuniperAlarmTask
+
+
+class TestJuniperalarmTask:
+    @pytest.mark.asyncio
+    async def test_juniper_alarm_runs_without_errors(self, snmpsim, juniper_alarm_task):
```

Comment:
Comment to all test names: Since the class name states that these are test for the juniper alarm task, the test names dont need to include too much context, so `test_task_runs_without_errors` or something like that should be enough. No need to state in class name and test names that its Juniper Alarm task specifically